src/OpenTelemetry.Instrumentation.AspNet/TracerProviderBuilderExtensions.cs

```diff
@@ -42,7 +42,9 @@ public static class TracerProviderBuilderExtensions
             var aspnetOptions = new AspNetInstrumentationOptions();
             configureAspNetInstrumentationOptions?.Invoke(aspnetOptions);
 
-            builder.AddDiagnosticSourceInstrumentation((activitySource) => new AspNetInstrumentation(activitySource, aspnetOptions));
+            builder.AddInstrumentation(() => new AspNetInstrumentation(aspnetOptions));
+            builder.AddSource(typeof(AspNetInstrumentation).Assembly.GetName().Name);
+            builder.AddLegacyActivityOperationName("Microsoft.AspNet.HttpReqIn");
```

Comment:
dont we create sibling activities in asp.net also?

Reply:
Thanks! I missed this one. I have added it.